### DISCUSSION TITLE
关于postcss配置的一些问题

### DISCUSSION THREAD

**Discussion by pisyell** [NervJS/taro#14414] [Q & A]
1. taro能不能通过postcss.config.js 这种文件式的去配置postcss？
2. 我该如何配置使用postcss-preset-env，因为我的项目用了tailwindCSS，里面的颜色rgba有问题，括号内被处理成了除法，我需要通过postcss再对其进行转换一下，适配低版本浏览器。
3. 如果问题1是可以的话，会不会影响'taro-plugin-tailwind'这个插件，还有就是文件和config文件中的h5.postcss的处理策略是什么？合并还是谁的优先级高？

**Comment by agileago** [comment]
应该可以直接配置 postcss.config.js即可，  tailwind 用 weapp-tailwindcss 这个插件  https://github.com/sonofmagic/weapp-tailwindcss
  
我这里有个模板  https://github.com/agileago/vue3-taro
